### ISSUE TITLE
Affects not fully displayed for spell slots by level

### ISSUE THREAD

**Issue by longhorn09** [longhorn09/lorebot#4]
```
Object 'bangle bracelet sky blue platinum decorated'
Item Type: WORN
Mat Class: metal        Material : platinum
Weight   : 2            Value    : 1000
Affects  : spell slots   by +1 at level 6
Affects  : spell slots   by +1 at level 8
Affects  : RESIST_MAGIC  by 5
Affects  : MEM_FACTOR    by 3
Effects  : DETECT_INVISIBLE
Restricts: NOMAGE NOTHIEF NOPALADIN NORANGER NOWARRIOR NODARK_KNIGHT NOBARBARIAN NOBLACK_ROBE NORED_ROBE NOWHITE_ROBE
Immune   : IMMUNE_COLD IMMUNE_ELECTRICITY
```
Issue initially identified by Pat and posted in Discord. Was able to duplicate and verify bug exists.

**Comment by longhorn09** [comment]
need to update regex, it's in formatAffects, regex assumes only + or - digits. In theory just change
```
  let affectBy = /([A-Za-z_\s]+)\s*by\s*([-+]?\d+)/;
```

to be more permissive

```
  let affectBy = /([A-Za-z_\s]+)\s*by\s*(.+)/;
```

regex match capture groups would remain unchanged

around line 259 in current commit

**Comment by longhorn09** [comment]
fixed in 322fff0
